chart/values.schema.json

```diff
@@ -100,6 +100,39 @@
                 }
             ]
         },
+        "securityContexts": {
+            "description": "Security context definition for the scheduler.",
```

Comment:
Sorry, probably could have been more clear with my original comments. This one should still be global, and what you had before was good. I mean the component specific descriptions needs to be updated to reflect they are scheduler/webserver/etc specific.

Reply:
Ah, yes I missunderstood. I think I should have looked at it properly as well. Tried to change it accordingly now.